gridcapa-core-cc-app/src/main/java/com/farao_community/farao/gridcapa_core_cc/app/configuration/AmqpMessagesConfiguration.java

```diff
@@ -25,10 +25,6 @@
 @Configuration
 public class AmqpMessagesConfiguration {
 
-    @Value("${core-cc-runner.bindings.response.destination}")
```

Comment:
Don't forget to remove those values from `test/resources/application.yml` in **gridcapa-core-cc** project and from `application.yml` and configmaps in **griddcapa-deployment** project